pkgs/development/tools/sqlint/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, bundlerApp, fetchurl }:
+
+let
+  gemset = import ./gemset.nix;
+  pgQueryVersion = gemset.pg_query.version;
+  LIB_PG_QUERY_TAG = "10-1.0.1";
+  libpgQuerySrc = fetchurl {
+    url = "https://codeload.github.com/lfittl/libpg_query/tar.gz/${LIB_PG_QUERY_TAG}";
+    sha256 = "0m5jv134hgw2vcfkqlnw80fr3wmrdvgrvk1ndcx9s44bzi5nsp47";
+  };
+in bundlerApp {
+  pname = "sqlint";
+  gemdir = ./.;
+  exes = [ "sqlint" ];
+
+  gemConfig = {
+    pg_query = attrs: {
+      #patches = [./dir.patch];
```

Comment:
I tried adding a patch to at least see whether the working directory actually is `$out/${attrs.ruby.gemPath}/gems/pg_query-${pgQueryVersion}/ext/pg_query`, but the patch is unable to find `extconf.rb`.